consensus/ibft/ibft.go

```diff
@@ -67,6 +68,9 @@ type Ibft struct {
 
 	// aux test methods
 	forceTimeoutCh bool
+
+	// Staking hub used for interacting with other modules
+	stakingHub *staking.StakingHub
```

Comment:
I don't see this used anywhere, but maybe I'm mistaken...

Reply:
You're right, this was leftover from a previous implementation.
Removing now 👍